epregressions/energyplus.py

```diff
@@ -38,13 +38,17 @@ def execute_energyplus(e_args: ExecutionArguments):
 
     # Save the current path so we can go back here
     start_path = os.getcwd()
+
+    std_out = b""
+    std_err = b""
+
     try:
 
         new_idd_path = os.path.join(e_args.test_run_directory, 'Energy+.idd')
         shutil.copy(idd_path, new_idd_path)
 
         # Copy the weather file into the simulation directory
-        if e_args.run_type != ForceRunType.DD:
+        if e_args.weather_file_name:
```

Comment:
As long as a weather file is provided (always...), copy it over, even on DD runs.